includes/wp-cli.php

```diff
@@ -78,17 +78,41 @@ function insert_redirect( $args, $assoc_args ) {
 		$inserted = WPCOM_Legacy_Redirector::insert_legacy_redirect( $from_url, $to_url );
 
 		if ( ! $inserted || is_wp_error( $inserted ) ) {
-			WP_CLI::warning( sprintf( "Couldn't insert %s -> %s", $from_url, $to_url ) );
+			WP_CLI::error( sprintf( "Couldn't insert %s -> %s", $from_url, $to_url ) );
```

Comment:
Why do we want to halt execution when an insert fails (there wasn't a reason noted in the commit message)? It might be annoying if the failure happens halfway through a really large list.

Reply:
This function is for the subcommand to insert only a single redirect. Without halting execution, the warning is printed, followed by the success message on line 84. Halting execution prevents the success message from displaying.